lib/manager/components/version/VersionRetrievalBadge.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import Icon from '@conveyal/woonerf/components/icon'
+import * as React from 'react'
```

Comment:
I don't think we need to import everything from React here. Seems like `React, {Component}` should suffice.

Reply:
I think that was leftover from when I was trying to specify a flow type of React.Node.